### DISCUSSION TITLE
whats possible with new Groove patterns?

### DISCUSSION THREAD

**Discussion by maks** [bangcorrupt/hacktribe#34] [Q&A] [answered]
Hi, @bangcorrupt  thanks so much for working on this project and making the patches available.

I was wondering how much is possible with new groove patterns, is it only possible to move the existing beats back/forwards for some microtiming around the beat quantisation, or is it possible to add re-triggering as well, for things like hihat rolls etc?

**Comment by BKLronin** [comment]
FYI I´m using the lag function(ifx) to place fe snares on the exact point I want them. I like this more than the patterns as you can tune it in more organically.

For repeats like trap hihats fe I simply use the repeat or another ifx and then automate it just to hit one hihat step for x amount of time.

Both have the downside of occupying the effect but sometimes its worth it.

**Comment by bangcorrupt** [comment, marked as answer]
Hi, no problem.  Groove templates allow altering trigger time, gate time and velocity by a certain amount for each step.  Re-triggering and tracker style effects aren't possible using groove templates, but if I can find the sequencing MFX (Odd Stepper, etc...) I may be able to edit them.  It would be good to separate these from the audio FX too, as I think they should be able to run at the same time without any DSP cost.

**Reply by maks** [reply]
Thanks so much for both your replies!
Yes I've used the delay technique before after seeing it on a YT video, but I found it a bit tricky to use and not having to use up the Ifx would be nice.

**Comment by maks** [comment]
I also wanted to add a big thank you for making this patch, its given my E2 a completely new life!
